docs/events/api/timeline-template-api-update.asciidoc

```diff
@@ -0,0 +1,385 @@
+
+==== Request body
+
+A JSON object defining the Timeline or Timeline template query and time filter.
+
+NOTE: For detailed information about the Timeline object schema and its
+corresponding UI components, see <<timeline-object-schema>>.
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`timeline` |<<timeline-object-schema, timeline>> a|The timeline object, which
+defines the search criteria and time range.
+
+When you are updating a Timeline template, provide these fields so you can
+easily <<timeline-api-import, import template>> updates:
+
+* `templateTimelineId`: Unique identifier (UUID, for Timeline templates only). This must match the `templateTimelineId` of an existing Timeline template that you're updating.
+* `templateTimelineVersion`: Template version number (for Timeline templates only). This must be a Timeline template version greater than the existing one.
+
+|Yes
+|`timelineId` |String |The `savedObjectId` of an existing Timeline that you're updating.
+|Yes
+|`version` |String |The version of an existing Timeline that you're updating.
+|Yes
+|==============================================
+
+===== Example requests
+
+*Example 1*
+
+Updates an existing Timeline.
+
+[source,console]
+--------------------------------------------------
+PATCH api/timeline
+{
+  "timeline": {
+    "columns": [
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "@timestamp"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "message"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "event.category"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "event.action"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "host.name"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "source.ip"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "destination.ip"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "user.name"
+      }
+    ],
+    "dataProviders": [],
+    "description": "",
+    "eventType": "all",
+    "filters": [],
+    "kqlMode": "filter",
+    "kqlQuery": {
+      "filterQuery": null
+    },
+    "title": "abd",
+    "dateRange": {
+      "start": 1587370079200,
+      "end": 1587456479201
+    },
+    "savedQueryId": null,
+    "sort": {
+      "columnId": "@timestamp",
+      "sortDirection": "desc"
+    },
+    "created": 1587468588922,
+    "createdBy": "casetester",
+    "updated": 1587468588922,
+    "updatedBy": "casetester",
+    "timelineType": "default"
+  },
+  "timelineId": "4bc294e0-3516-11ee-9f62-49614d8a84fd", <1>
+  "version": "WzE5MTUsMV0=" <2>
+}
+--------------------------------------------------
+
+<1> `timelineId` must match the existing Timeline's `savedObjectId`.
+<2> `version` must match the existing Timeline version.
```

Comment:
Hm, this explanation is still a little unclear to me. @angorayc would you mind verifying this?
 
```suggestion
<2> `version` must match the version of the Timeline that you're updating.
```

Reply:
For example, you want to update the title of an existing timeline

```
{
  "timeline": {
    ....
    "title": "abd",
    ....
  },
  "timelineId": "4bc294e0-3516-11ee-9f62-49614d8a84fd", <1>
  "version": "WzE5MTUsMV0=" <2>
}
```

In your request, just  leave timelineId and version as it is and update the title to the new value.